gigascience/gigadb-website
# Update GigaDBUpload-template_v18.xls

fix: updated the CV tab with new attributes from database attribute table

# Pull request for issue: no issue ticket created

This is a pull request for the following functionalities:

* enables spreadsheet users to utilise new valid attributes in the sample table

## How to test?
Enter one of the new attribute names (e.g. alt_acc_GSA_sample) in the blue cells of the heading row of the sample table
see the grey cell in row 3 under the one you have just edited change to show the relevant description of that attribute
